test6.js

```diff
@@ -0,0 +1,24 @@
+
+
+
+var potato = "";
+
+if (potato === "potato") {
+  console.log("hey you");
+}
+
+
+
+
+var foo = '' ;
+
+if(foo  === "bar") {
+  console.log( 'hey' )
+}
+
+
+var foo77 = '' ;
+
+if(foo77  === "baaaaaaaaaar") {
+  console.log( 'heaaaaaaay' )
```

Comment:
Codacy has a fix for the issue: [Replace `·'heaaaaaaay'·)` with `"heaaaaaaay");`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5796)
```suggestion
  console.log("heaaaaaaay");
```